lib/myapp18_web/user_auth.ex

```diff
@@ -98,13 +110,21 @@ defmodule Myapp18Web.UserAuth do
   end
 
   @doc """
-  Authenticates the user by looking into the session
-  and remember me token.
+  Authenticates the user by looking into the session and remember me token.
+
+  Will reissue the session token if it is older than the configured age.
   """
   def fetch_current_scope_for_user(conn, _opts) do
-    {user_token, conn} = ensure_user_token(conn)
-    user = user_token && Accounts.get_user_by_session_token(user_token)
-    assign(conn, :current_scope, Scope.for_user(user))
+    with(
+      {token, conn} <- ensure_user_token(conn),
+      {user, token_inserted_at} <- Accounts.get_user_by_session_token(token)
+    ) do
```

Comment:
Nitpick, sorry, but we do use this style on Phoenix:

```suggestion
    with {token, conn} <- ensure_user_token(conn),
         {user, token_inserted_at} <- Accounts.get_user_by_session_token(token) do
```